config/locales/ru.yml

```diff
@@ -93,8 +96,134 @@ ru:
     edit:
       title: "Редактирование сведений об организации"
   date:
```

Comment:
may be move date and time localization in a separate file, example date_time.ru ?